### PR TITLE
Refactor(dev experience) speed up dev mode

### PR DESCRIPTION
**Pull request by GailMelanie**
# Description

Please include a summary of the changes and link the related github issue.
Please include relevant motivation and context.

Fixes # (Issue)

## Type of change

Please delete options that are not relevant.

-   [ ] Minor change (non-breaking change, e.g. documentation adaption)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)

# Checklist:

-   [ ] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] I have added tests that prove my fix or my feature works
-   [ ] New and existing tests pass locally with my changes
-   [ ] My changes contain no console logs

### CODE REVIEW

File: src/app/[locale]/viewer/_components/submodel/carbon-footprint/visualization-components/ProductJourney.tsx [hunk 1]
```diff
@@ -118,19 +117,18 @@ function getMarkerLayers(coordinatesPerLifeCyclePhase: AddressPerLifeCyclePhase[
...
-
+            const markerIconName = `LocationMarker_${phase.lifeCyclePhase}`;
+            
             return new VectorLayer({
                 source: markerSource,
                 style: new Style({
                     image: new Icon({
                         anchor: [0.5, 1],
-                        src: productJourneyIconName ? productJourneyIconName.src : '',
+                        src: `/LocationMarkers/${markerIconName}.svg`,
```
Comment (GailMelanie) [inline src/app/[locale]/viewer/_components/submodel/carbon-footprint/visualization-components/ProductJourney.tsx:127]: With this changes, --turbo works and is way faster than the regular 'next dev'. The downside is, that we can no longer use the svg?url solution we had before. I tried to find a quick solution but I'm not convinced 😄

File: next.config.js [hunk 2]
```diff
@@ -7,6 +7,16 @@ const withNextIntl = createNextIntlPlugin();
 const nextConfig = {
     output: 'standalone', // Outputs a Single-Page Application (SPA).
     distDir: './dist', // Changes the build output directory to `./dist/`.
+    experimental: {
+        turbo: {
+            rules: {
+                '*.svg': {
+                    loaders: ['@svgr/webpack'],
```
Comment (GailMelanie) [inline next.config.js:14]: The Issue with turbopack ist, that our webpack config below doesn't apply. And I couldn't find a way to set the rules for the *.svg?url....

File: src/lib/util/ProductJourneyIcon.spec.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import fs from 'fs'
```
Comment (GailMelanie) [inline src/lib/util/ProductJourneyIcon.spec.ts:1]: Open to suggestions where to put test files which are not related to one specific file/folder 😄

Comment (JonathanXITASO) [reply src/lib/util/ProductJourneyIcon.spec.ts:1]: are those still unit tests then?
maybe even put them into its separate subdirectory /test/...

**Review by JonathanXITASO — APPROVED** [review]
